Files/Views/ModernShellPage.xaml.cs

```diff
@@ -255,10 +255,11 @@ private void SidebarControl_SidebarItemInvoked(object sender, Controls.SidebarIt
                     AssociatedTabInstance = this
                 },
                 new SuppressNavigationTransitionInfo());
+            ReloadColumnView?.Invoke(this, EventArgs.Empty);
 
             NavigationToolbar.PathControlDisplayText = FilesystemViewModel.WorkingDirectory;
         }
-
+        public static event EventHandler ReloadColumnView;
```

Comment:
Why would we need this event? BaseLayout-derived layout modes should load the new content upon being navigated to without the need for something like this in the shell page. 🙂

Reply:
@TyJOrtiz Why didn't the OnNavigatedTo event work out? You should be able to override it iirc

Reply:
Ok, I must've done something wrong there. I added the override and it works

Reply:
Excellent!